canonical-web-and-design/multipass.run
# Update homepage and remove install page

## Done
- Updated the homepage to match the copy docs
- Removed the /install page
- Cleaned out the install assets

## QA
- Go to the homepage
- Check it matches the [copy doc](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit#heading=h.5s7zia3ks9ng)

Fixes https://github.com/canonical-web-and-design/multipass.run/issues/32
